CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -1,20 +1,48 @@
-==============================
-Repair Tree
-==============================
+    As engineers practice their craft, they learn certain processes. These are 
+listed below. 
+
+Energy:
+    Most procedures that an engineer can perform in the field requires the 
+expenditure of a non-trivial amount of energy. During combat, an engineer may 
+overcharge their standard issue batteries as outlined in the Engineer section of 
+the 02_classes.txt document. When not in combat, energy is vented from the 
+batteries down to their safe storage limit. If the batteries were to remain 
+overcharged for longer than the duration of an encounter, they would burn out or 
+burn up and explode depending on the limit. The max limit and safe storage limit 
+are outlined in the "02_classes.txt" document in Character Creation. 
+
+Materials:
+    An engineer will often use an extruder or 3D printer in the field to 
+construct helpful things like drones or cover in the heat of combat. To do this, 
+the Engineer uses a resource called Material. An engineer's extruder has a 
+built-in recycler which can be used to salvage usable material from a variety of 
+sources. Often, recycling spent, unused or damaged parts is part of the standard 
+procedure of a given process. Material accrued on mission persists between 
+encounters and over small rests.
+
+Munitions:
+    Munitions are used in the engineer's extruder to print more ... active 
+components. Due to regulations munitions extrusion material is not retained 
+between combat encounters. 
+
 Network Motes: 
-    Any repair process that an engineer uses on a cybernetic ally
-(even if that process doesn't specify that it can only be used on cybernetic
-allies) leaves a "network mote" on that ally. When calculating roaming range
-of drones built in the build tree, the range of the drone is taken from the
-nearest network mote or the engineer that deployed it.
+    Any repair process that an engineer uses on a cybernetic ally (even if that 
+process doesn't specify that it can only be used on cybernetic allies) leaves a 
+"network mote" on that ally. When calculating roaming range of drones built in 
+the build tree, the range of the drone is taken from the nearest network mote or 
+the engineer that deployed it.
 
-    Unlocking a Repair tree skill requires having unlocked the skill below it on 
+    Unlocking a Process requires having unlocked the Process below it on Skill 
 the tree. Please see the Engineer Skill Tree - Repair.png for dependencies.
 
+=================================
+=====		Repair Tree		=====
+=================================
```

Comment:
If this will parse "=====\t\tRepair Tree	\t\t=====" as a chapter heading then this is not a blocker, but I question the necessity. Of note, the Build tree chapter heading below is much more normal, so this is an inconsistency.

Reply:
nope, you're correct. That's totally non-compliant. Nuking it.